pontoon/contributors/static/css/settings.css

```diff
@@ -83,10 +83,12 @@
   margin-top: 10px;
 }
 
-#email-warning {
-  display: none;
+#profile-form .help {
+  color: #FED271;
+  font-style: italic;
+  font-weight: bold;
```

Comment:
Now that the text is always visible, I find the styling a little too intrusive.

I'd remove bold (bold+italic at the same time is rarely a good idea) and change color to `#888888`. Please also add `margin-top: 5px;`.